.github/workflows/update-universes.yml

```diff
@@ -0,0 +1,86 @@
+name: Update universes
+
+on:
+  schedule:
+    - cron: '27 7 * * *'
+  # FIXME: remove
+  push:
+    branches:
+      - feat/ci
+
+jobs:
```

Comment:
Crazy that you can't factorize any of this :raised_eyebrow:  But we can always come back to this later if we ever figure it out...